src/foam/util/filesystem/FileWalker.js

```diff
@@ -0,0 +1,120 @@
+foam.CLASS({
+  package: 'foam.util.filesystem',
+  name: 'FileWalker',
+  topics: ['files', 'directoryStart', 'directoryEnd', 'error'],
+  documentation: `
```

Comment:
Just a suggestion/personal preference, but I find it easier to scan a model file when there are empty lines between axioms (except for package, name and extends).